src/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+require_relative './rental'
+
+class Book
+  attr_accessor :date, :title, :rental
```

Comment:
- [ ] Kindly notice that the `author` attribute should be `accessor`, not the `rental`.
- [ ] Also remove the `date` from `accessor` in the `Book` class.

```ruby
attr_accessor :title, :author
```